code-of-conduct.md

```diff
@@ -1,23 +1,27 @@
+Welcome to Woodcrest, a community where diversity, creativity, and respect form the bedrock of our shared experience. As members of this vibrant world, it is crucial that we adhere to a code of conduct and sim rules that uphold the integrity and enjoyment of all participants. Here is our comprehensive guide to ensuring a positive, inclusive, and enriching environment for everyone:
 
-1. **Foster Openness**: Engaging in local chat helps include everyone. While private messages (IMs) have their place, especially for welcoming those less inclined to speak up, always respect privacy and consent to avoid harassment.
+1. **Embrace and Respect Diversity**: Woodcrest is a diverse mosaic of roleplay experiences and backgrounds. We champion inclusivity and encourage all members to lead by example, fostering an environment where guidance is given with respect, not elitism.
 
-1. **Honor Boundaries**: Consent is key in roleplay. Avoid forcing unwelcome scenarios on others. Public events require participation by presence, but remember to seek approval for any significant environmental impact. Keep sensitive roleplay private and use discretion in public spaces to ensure comfort for all.
+1. **Communication is Key**: Open and respectful communication, primarily through local chat, is encouraged to maintain inclusivity. While private messages have their place, consent and privacy must always be respected to prevent harassment.
 
-1. **Minimize Drama**: Approach conflicts constructively and respect the community by avoiding slander and toxicity. Strive for peaceful resolutions and maintain a positive environment.
+1. **Consent and Boundaries**: Consent is the cornerstone of interaction here. Avoid imposing scenarios on others without their agreement. Public roleplay should not infringe on community comfort, and sensitive content must be approached with discretion. We strictly prohibit excessive violence and discriminatory language, both in public and private settings.
 
-1. **Enhance, Don’t Replace**: Use shouts, dances, and gestures to complement, not overshadow, roleplay. Excessive or disruptive gestures, along with out-of-character shouts, are not permitted. Griefing, trolling, and unsolicited promotions are grounds for disciplinary action.
+1. **Drama and Disruptions**: Constructive conflict resolution is essential. Slander, toxicity, and behaviors that disturb the peace are not tolerated. Scheduled events should not be interrupted without prior consent, emphasizing respect for the planning and effort involved.
 
-1. **Respect Events**: Do not interrupt scheduled events without prior agreement with the organizers. Understand the effort behind event planning and maintain a collaborative spirit.
+1. **Roleplay Etiquette**: Enhancements like shouts and gestures should support, not overshadow, roleplay. We discourage any form of griefing, trolling, and spamming, as these actions detract from our collective experience.
```

Comment:
> gestures

I'm always torn on this in an IC sense. People use this _really_ badly and I can't think of many situations where they are appropriate.